### PR TITLE
Docs adjustments with latest changes

### PR DESCRIPTION
**Pull request by sdurrheimer**
@juliusv @fabxc

### CODE REVIEW

File: content/docs/introduction/faq.md [hunk 1]
```diff
@@ -79,15 +79,29 @@ Prometheus is released under the
...
+Yes, sending SIGHUP to the Prometheus process will reload
+and apply the configuration file. The different components attempt
+to handle failing changes gracefully.
+
 ### Can I send alerts?
 
 Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertmanager).
-[PagerDuty](http://www.pagerduty.com/) and email are supported.
+
+Currently the following external services are supported :
```
Comment (fabxc) [inline content/docs/introduction/faq.md]: Comma after currently. No space before colon.

File: content/docs/introduction/roadmap.md [hunk 2]
```diff
@@ -27,20 +27,11 @@ GitHub issue: [#9](https://github.com/prometheus/prometheus/issues/9)
 
 Currently Prometheus supports configuring static HTTP targets, as well as
 discovering targets dynamically via [DNS SRV
-records](http://en.wikipedia.org/wiki/SRV_record). We plan to support more
-types of service discovery (e.g. Consul or Zookeeper) in the future. Some will
+records](http://en.wikipedia.org/wiki/SRV_record) and [Consul](https://www.consul.io/). There is also a file-based interface that allows you to connect your own discovery mechanisms. We plan to support more
+types of service discovery (e.g. Zookeeper) in the future. Some will
```
Comment (fabxc) [inline content/docs/introduction/roadmap.md]: The last sentence is solved by the file-based SD.

File: content/docs/introduction/faq.md [hunk 3]
```diff
@@ -79,15 +79,29 @@ Prometheus is released under the
 After extensive research it has been determined that the correct plural of
 'Prometheus' is 'Prometheis'.
 
+### Can I reload Prometheus configuration ?
```
Comment (fabxc) [inline content/docs/introduction/faq.md]: Prometheus's.

No space before the question mark.

**Comment by fabxc** [comment]
Looks good otherwise :+1: 
Thanks for finding all the outdated stuff!

File: content/docs/introduction/faq.md (hunk 1 above)
Comment (sdurrheimer) [inline content/docs/introduction/faq.md]: "systems" sounds fine. I will make the change.

**Comment by sdurrheimer** [comment]
There might be others. I will keep a eye on that.

**Comment by sdurrheimer** [comment]
We might need to write something about deleting Collected data into the FAQ. This question is often asked on IRC.

**Comment by juliusv** [comment]
Thanks a bunch!

**Pull request merged**